.circleci/config.yml

```diff
@@ -4,11 +4,10 @@ jobs:
     # `dep ensure` must be run from within a known GOPATH/src.
     working_directory: /go/src/github.com/helm/chart-releaser
```

Comment:
Nice. The working directory doesn't matter now with go modules, right?

Reply:
Yes, but speaking of with, we should remove it so we are not inside GOPATH. Otherwise, we'd have to set GO111MODULE=off.